### PR TITLE
Update link URLs

### PR DESCRIPTION
**Pull request by zseemore**
These changes include updating links that were not working within the notebooks. Also updated the 1-object-detection notebook with the code for displaying image properties.

Please let me know if you need more information.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: rekognition-apis/1-object-detection.ipynb [hunk 1]
```diff
@@ -665,7 +781,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.8.6"
+   "version": "3.7.10"
```
Comment (matteofigus) [inline rekognition-apis/1-object-detection.ipynb]: Same here

File: rekognition-apis/1-object-detection.ipynb [hunk 2]
```diff
@@ -650,10 +765,11 @@
   }
  ],
  "metadata": {
+  "instance_type": "ml.t3.medium",
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 3 (Data Science)",
    "language": "python",
-   "name": "python3"
+   "name": "python3__SAGEMAKER_INTERNAL__arn:aws:sagemaker:us-east-1:081325390199:image/datascience-1.0"
```
Comment (matteofigus) [inline rekognition-apis/1-object-detection.ipynb]: Do we need these changes in the notebook's metadata? If we don't, I would prefer to revert those to keep this notebook consistent with the others.

File: rekognition-apis/1-object-detection.ipynb [hunk 3]
```diff
@@ -85,7 +91,7 @@
     "# We will only use this directory to download images from S3 bucket and draw bounding boxes\n",
     "\n",
     "!mkdir m1tmp\n",
-    "tempFolder = 'm1tmp/'"
+    "tempFolder = 'm1tmp'"
```
Comment (matteofigus) [inline rekognition-apis/1-object-detection.ipynb]: This variable seems to be unused, perhaps we can just delete this line?

```suggestion
```

**Review by matteofigus — CHANGES_REQUESTED** [review]
A couple of nit picks

File: rekognition-apis/1-object-detection.ipynb (hunk 2 above)
Comment (zseemore) [reply rekognition-apis/1-object-detection.ipynb]: No, we do not need these changes

File: rekognition-apis/1-object-detection.ipynb (hunk 3 above)
Comment (zseemore) [reply rekognition-apis/1-object-detection.ipynb]: Yes, we can delete this variable as it is unused

**Review by matteofigus — APPROVED** [review]
LGTM

File: rekognition-apis/1-object-detection.ipynb [hunk 4]
```diff
@@ -150,7 +169,10 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "scrolled": true,
```
Comment (ConnorKirk) [inline rekognition-apis/1-object-detection.ipynb:173]: What does this line do?

Comment (matteofigus) [reply rekognition-apis/1-object-detection.ipynb:173]: I found some info in a [thread](https://github.com/jupyterlab/jupyterlab/issues/4028) - I guess it's mostly styling related for getting some long output automatically scrolled to bottom - some arguing it should be the default. I guess I don't think I have a strong opinion.

Comment (ConnorKirk) [reply rekognition-apis/1-object-detection.ipynb:173]: Perfect, thanks. I looked for docs on the attribute but couldn't find anything.

**Review by ConnorKirk — APPROVED** [review]
LGTM. Thank you for the contribution!

**Pull request merged**